tests/integration/backend_test.py

```diff
@@ -999,3 +999,178 @@ def test_scratch_removal(authenticated_quum_backend: QuantinuumBackend) -> None:
     shots = r.get_shots()
     assert len(shots) == 3
     assert all(len(shot) == 5 for shot in shots)
+
+
+@pytest.mark.skipif(skip_remote_tests, reason=REASON)
+@pytest.mark.parametrize(
+    "authenticated_quum_backend", [{"device_name": "H1-1E"}], indirect=True
+)
+@pytest.mark.parametrize(
+    "language",
+    [
+        Language.QASM,
+        # https://github.com/CQCL/pytket-quantinuum/issues/236
```

Comment:
I would assume that this one also needs to wait for the wasm implementation (see https://github.com/CQCL/pytket-quantinuum/issues/232 )